listenbrainz/db/timescale.py

```diff
@@ -8,7 +8,7 @@
 from listenbrainz import config
 
 # This value must be incremented after schema changes on replicated tables!
-SCHEMA_VERSION = 5
+SCHEMA_VERSION = 6
```

Comment:
I think we also need to add an empty `''` for email like we do for auth_token to https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/db/dump.py#L58-L68